src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -115,6 +125,32 @@ protected Endpoint getEndpoint() {
         return Endpoint.INTERNALUSERS;
     }
 
+    @Override
+    protected void handleGet(final RestChannel channel, RestRequest request, Client client, final JsonNode content) throws IOException {
+        final String resourcename = request.param("name");
+
+        SecurityDynamicConfiguration<?> configuration = load(getConfigName(), true);
+        filter(configuration);
+
+        String filterBy = request.param("filterBy", "all");
```

Comment:
Default value of all, e.g. no results?  IMO this should be nullable

Reply:
FilterBy "service" will return Service Accounts
FilterBy "Internal" will return Internal Accounts
The default "All" will return All Accounts

Reply:
I made the change, but wouldn't "Include" be read as "add" and cause confusion as well?